torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5521,6 +5521,22 @@ def sample_inputs_softmax_variant(op_info, device, dtype, requires_grad, with_dt
     ]
 
 
+def sample_inputs_masked_softmax(op_info, device, dtype, requires_grad, with_dtype=False, **kwargs):
+    """Sample inputs for masked softmax and log_softmax.
+
+    Masked reduction operator is a reduction operator with trailing
```

Comment:
nit: do you mean "masked normalization"?

Reply:
yes, fixed.